pages/discover.js

```diff
@@ -251,6 +252,7 @@ const DiscoverPage = ({ router, intl }) => {
                           setRouteParam('sort', selected && selected.key);
                         }
                       }}
+                      minwidth={space}
```

Comment:
Please see https://styled-system.com/api/#layout, this is not the correct spelling for this prop.

Also can you add screenshots of the changes to the pull request?

Reply:
Ok. will work to that